ui/dashboards/src/components/PanelGroupDialog/PanelGroupDialog.test.tsx

```diff
@@ -42,7 +42,7 @@ describe('Add Panel Group', () => {
     const storeApi = renderDialog();
 
     // Open the dialog for a new panel group
-    act(() => storeApi.getState().openPanelGroupDialog());
+    act(() => storeApi.getState().addPanelGroup());
```

Comment:
thinking we should figure out how to open the dialog without using the store api here

Reply:
Yeah, I was thinking we should figure out if there's a way to make these test work without manipulating the store directly.